demos/python_demos/multi_camera_multi_person_tracking/run.py

```diff
@@ -0,0 +1,146 @@
+"""
```

Comment:
The main script of the demo should have the same name as the demo itself, for consistency with other demos.